website/package.json

```diff
@@ -48,10 +48,10 @@
     ]
   },
   "dependencies": {
-    "@docusaurus/core": "^2.4.1",
-    "@docusaurus/plugin-google-gtag": "^2.4.1",
-    "@docusaurus/plugin-pwa": "^2.4.1",
-    "@docusaurus/preset-classic": "^2.4.1",
+    "@docusaurus/core": "0.0.0-5658",
```

Comment:
Are these version numbers correct? 🤔 
when are we going to move to 3.y.z?

Reply:
@cipolleschi those are canary Docusaurus versions: https://docusaurus.io/community/canary

As soon as this PR is merged, I'll release Docusaurus v3-beta.0 and we can replace those with the first beta.

Or if you agree to merge, I can also publish v3-beta.0 first, and update this PR before merging.

My goal here is to be able to say to the community that a few major Docusaurus sites are already running on Docusaurus v3 beta (Docusaurus + Jest + React-Native), showing it is safe, and encourage adoption. 

I don't consider those canaries to be unstable myself, and could eventually release this canary as v3 as GA today. This is mostly a way to reassure the Docusaurus community and help increase adoption so that we can catch the few minor bugs that we might have missed. Similarly to when RN releases a RC and is looking for community feedback ;)

Reply:
Thanks for the explanation. I think that they are fine as they are.